third_party/move/testing-infra/transactional-test-runner/src/vm_test_harness.rs

```diff
@@ -407,43 +425,43 @@ static PRECOMPILED_MOVE_STDLIB: Lazy<FullyCompiledProgram> = Lazy::new(|| {
     )
     .unwrap();
     match program_res {
-        Ok(stdlib) => stdlib,
+        Ok(stdlib) => Some(stdlib),
         Err((files, errors)) => {
             eprintln!("!!!Standard library failed to compile!!!");
             move_compiler::diagnostics::report_diagnostics(&files, errors)
         },
     }
 });
 
-static MOVE_STDLIB_COMPILED: Lazy<Vec<CompiledModule>> = Lazy::new(|| {
-    let (files, units_res) = move_compiler::Compiler::from_files(
-        move_stdlib::move_stdlib_files(),
-        vec![],
-        move_stdlib::move_stdlib_named_addresses(),
-        Flags::empty().set_skip_attribute_checks(true), // no point in checking here.
-        KnownAttribute::get_all_attribute_names(),
-    )
-    .build()
-    .unwrap();
-    match units_res {
-        Err(diags) => {
-            eprintln!("!!!Standard library failed to compile!!!");
-            move_compiler::diagnostics::report_diagnostics(&files, diags)
-        },
-        Ok((_, warnings)) if !warnings.is_empty() => {
-            eprintln!("!!!Standard library failed to compile!!!");
-            move_compiler::diagnostics::report_diagnostics(&files, warnings)
-        },
-        Ok((units, _warnings)) => units
-            .into_iter()
-            .filter_map(|m| match m {
-                AnnotatedCompiledUnit::Module(annot_module) => {
-                    Some(annot_module.named_module.module)
-                },
-                AnnotatedCompiledUnit::Script(_) => None,
-            })
-            .collect(),
+pub struct PrecompiledFilesModules(Vec<String>, Vec<AnnotatedCompiledUnit>);
+
+impl PrecompiledFilesModules {
+    pub fn new(files: Vec<String>, modules: Vec<AnnotatedCompiledUnit>) -> Self {
```

Comment:
I am assuming `files` are vector of strings because of the type plumbing, but seems like they might be better as vector of paths (from just seeing this struct). This is a nit.